scripts/check_files_compliance.py

```diff
@@ -110,11 +110,16 @@ def ignore(root, file):
     if '.idea' in root:
         return True
 
+    # Apib files have an "inline" license, so they are ignored
     if file.endswith('.apib'):
-      return True
+        return True
+
+    # Markdown files are ignored
+    if file.endswith('.md'):
+        return True
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/8e1afd356293aa678c229fc428fc3140088e8280